pkg/activator/handler/metrics.go

```diff
@@ -59,6 +63,12 @@ func register() {
 			Aggregation: view.LastValue(),
 			TagKeys:     []tag.Key{metrics.PodKey, metrics.ContainerKey},
 		},
+		&view.View{
+			Description: "The number of requests that the Activator handles with TLS",
+			Measure:     tlsConnectionCountM,
+			Aggregation: view.Count(),
+			TagKeys:     []tag.Key{metrics.PodKey, metrics.ContainerKey, metrics.ResponseCodeKey, metrics.ResponseCodeClassKey},
```

Comment:
If you take my tagging suggestion, You'd need to add the SecurityMode to `requestCountM`.  I'd add it to `responseTimeInMsecM` as well (and maybe `requestConcurrencyM`), to allow comparing request latency for TLS and non-TLS if we decide to do that some day.